joy_lin/alpha_challenges/string_manipulation.md

```diff
@@ -19,13 +19,18 @@ iverson_rant = "If I can't practice, I can't practice, man. If I'm hurt, I'm hur
 Programmatically manipulate `iverson_rant` to reflect the output above.
 
 ```ruby
-# write your code here
+rant_words = iverson_rant.split(" ")
+first_letter = rant_words.map do |word|
+  word[0]
```

Comment:
There's actually a cool built-in method `chr` for getting the first character of a string (which is the same as your word[0]).
I.e:
```
my_string = 'cat in the hat'
my_string.chr => 'c'
```

Reply:
Wow, got it!